python/tskit/trees.py

```diff
@@ -5670,15 +5670,25 @@ def trait_correlation(self, W, windows=None, mode="site", span_normalise=True):
             span_normalise=span_normalise,
         )
 
-    def trait_regression(
+    def trait_regression(self, *args, **kwargs):
+        """
+        Deprecated synonym for :meth:`trait_linear_model`.
+        """
+        warnings.warn(
+            "This is deprecated: please use trait_linear_model( ) instead.",
+            FutureWarning,
+        )
+        return self.trait_linear_model(*args, **kwargs)
+
+    def trait_linear_model(
         self, W, Z=None, windows=None, mode="site", span_normalise=True
     ):
         """
-        For each trait w (i.e., each column of W), performs the least-squares
-        linear regression :math:`w ~ g + Z`,
+        For each trait w (i.e., each column of W), finds the coefficient of genotype
```

Comment:
Are we missing some words around "genotype" here?

Reply:
Well, I meant "the coefficient of :math:`g` in...", and sometimes `g` is genotype. I'll change it.